pypit/arsave.py

```diff
@@ -471,9 +477,40 @@ def save_2d_images(slf, clobber=True):
     -------
 
     """
+    import datetime
+    # Original header
+    scidx = slf._idx_sci[0]
+    path = fitsdict['directory'][scidx]
+    ifile = fitsdict['filename'][scidx]
+    headarr = [None for k in range(settings.spect['fits']['numhead'])]
+    for k in range(settings.spect['fits']['numhead']):
+        headarr[k] = pyfits.getheader(path+ifile, ext=settings.spect['fits']['headext{0:02d}'.format(k+1)])
+
     # Primary header
     prihdu = pyfits.PrimaryHDU()
+    # Update with original header, skipping a few keywords
     hdus = [prihdu]
+    hdukeys = ['BUNIT', 'COMMENT', '', 'BITPIX', 'NAXIS', 'NAXIS1', 'NAXIS2',
```

Comment:
I think there is, but can't recall why.  I guess just to remove header cards
that are empty.  So maybe not necessary but nice to have.